### PR TITLE
fix: case-sensitivity in apps path

### LINKED ISSUE #6548: [CAL-820] lowercase apps: cal.com/apps/Signal 404 (whereas /signal works)
<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-820](https://linear.app/calcom/issue/CAL-820/lowercase-apps-calcomappssignal-404-whereas-signal-works)</sub>

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #6548 

**Environment**: Staging(main branch) / Production

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by JeroenReumkens** [comment]
These pages are also indexed for SEO, right @PeerRich?  In that case we now have multiple URL's that could end up in Google. So it would be good to add a conical meta tag in that case, which tells google that there's only one single source of truth for every app page.

**Comment by PeerRich** [comment]
> These pages are also indexed for SEO, right @PeerRich? In that case we now have multiple URL's that could end up in Google. So it would be good to add a conical meta tag in that case, which tells google that there's only one single source of truth for every app page.

it actually turns out we have "no-index, no-follow" in our meta tag which means google never indexed them

https://github.com/calcom/cal.com/pull/4943/files#diff-3042eafd6ef787392d368974c4283d202f74a510aa121a4fc8bfc892ceb2d893R312-R313

we should remove this

**Review by PeerRich — APPROVED** [review]

File: apps/web/pages/apps/[slug]/index.tsx [hunk 1]
```diff
@@ -53,15 +53,15 @@ export const getStaticPaths: GetStaticPaths<{ slug: string }> = async () => {
 
   return {
     paths,
-    fallback: false,
+    fallback: "blocking",
```
Comment (PeerRich) [inline apps/web/pages/apps/[slug]/index.tsx:56]: what does this do?

Comment (G3root) [reply apps/web/pages/apps/[slug]/index.tsx:56]: all the paths returned by `getStaticPaths` are lower cased. with the fallback false, slug with the title case throws an 404 error. while fallback is blocking, it runs `getStaticProps` and check if it's matches any db record.

**Review by emrysal — APPROVED** [review]

**Pull request merged**